doc/whats-new.rst

```diff
@@ -16,9 +16,42 @@ What's New
 v0.7.2 (unreleased)
 -------------------
 
+This is a bug fix release that includes two small, backwards compatible enhancements.
+We recommend that all users upgrade.
+
 Enhancements
 ~~~~~~~~~~~~
 
+- Rolling window operations on DataArray objects are now supported via a new
+  :py:meth:`xarray.DataArray.rolling` method.
+
+  .. ipython::
+    :verbatim:
+
+    In [1]: import xarray as xr; import numpy as np
+
+    In [2]: x = xr.DataArray(np.random.random((3, 5)))
```

Comment:
I would opt for an easier to understand example -- possibly with sensibly labeled dimensions (e.g., `'x'`, `'`y'`) -- and with non-random values (i.e., from range or linspace).